docusaurus/docs/Android/04-compose/05-message-components/03-message-list.mdx

```diff
@@ -166,6 +166,7 @@ We allow for a few ways of customizing the `MessageList` component, as per the s
 fun MessageList(
 	..., // State and actions
     modifier: Modifier = Modifier,
+    contentPadding: PaddingValues = PaddingValues(vertical = DefaultMessagesContentPadding),
```

Comment:
I would remove the `vertical = ...` part and just put:

`contentPadding: PaddingValues = PaddingValues(...)`

So that in case we change it up we don't have to update the docs